MOBILE_API.xml

```diff
@@ -5829,17 +5847,26 @@
         <param name="rpm" type="Boolean" mandatory="false">
             <description>The number of revolutions per minute of the engine</description>
         </param>
-        <param name="fuelLevel" type="Boolean" mandatory="false">
-            <description>The fuel level in the tank (percentage)</description>
+        <param name="fuelLevel" type="Boolean" mandatory="false" deprecated="true" since="6.2">
```

Comment:
```suggestion
        <param name="fuelLevel" type="Boolean" mandatory="false" deprecated="true" since="7.0">
```
Please make similar version changes for this parameter in the other vehicleData RPCs.